decentraland/unity-renderer
# Feat: Added UI blur resources.

## What does this PR change?



Added UI Blur resources added to the project and added the required render feature to the forward renderer file

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch={branch_name}&{desired_url_params}
2. Check UI and 3D objets are rendered as usual

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
